apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -972,6 +972,80 @@ const Ticket = new GQLListSchema('Ticket', {
         delete: false,
         auth: true,
     },
+    kmigratorOptions: {
+        indexes: [
+            {
+                type: 'BTreeIndex',
```

Comment:
We already have FK indexes. Do you have some proofes that it can help?

Reply:
removed it